src/strings-in-block/class-html.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace WPML\PB\Gutenberg\StringsInBlock;
+
+class HTML extends Base {
```

Comment:
This class is the transposition of the old class `WPML_Gutenberg_Strings_In_Block`. And I moved some shared code inside the `Base` class.